
Change filter variable in one of the Makefiles for the BF6064X-T

Index: OpenNetworkLinuxv2/packages/platforms/stordis/x86-64/bf6064x-t/onlp/builds/Makefile
===================================================================
--- OpenNetworkLinuxv2.orig/packages/platforms/stordis/x86-64/bf6064x-t/onlp/builds/Makefile
+++ OpenNetworkLinuxv2/packages/platforms/stordis/x86-64/bf6064x-t/onlp/builds/Makefile
@@ -1,2 +1,2 @@
-FILTER=src
+FILTER=x86_64_stordis_bf6064x_t
 include $(ONL)/make/subdirs.mk
